packages/oui-inline-adder/README.md

```diff
@@ -0,0 +1,167 @@
+# oui-inline-adder
+
+<component-status cx-design="complete" ux="complete"></component-status>
+
+## Installation
+
+```less
+@import 'oui-inline-adder/inline-adder';
+```
+
+## Usage
+
+### Normal
+
+```html:preview
+<div class="oui-inline-adder">
+  <div class="oui-inline-adder__item">
+    <div class="oui-inline-adder__row">
+      <div style="flex: 1 1 auto; padding: 1rem; padding-bottom: 0;">
```

Comment:
No inline style in our examples. Why not create a class these col/cell (with a modifier if necessary) ? :)

Reply:
This is done. Thanks for the suggestion :)